action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   payload-file-path: # path to JSON payload to send to Slack via webhook
     description: 'path to JSON payload to send to Slack if webhook route. If not supplied, json from GitHub event will be sent instead. If payload is provided, it will take preference over this field'
     required: false
+  use-file-as-valid-json:
```

Comment:
I'm thinking it might be better to use `payload-file-path-parsed` with a default `true` value here instead to match the `payload-file-path` parameter. Open to your thoughts on this though!